### DISCUSSION TITLE
Can I delete branches f0004 and f0008?

### DISCUSSION THREAD

**Discussion by adesutherland** [adesutherland/CREXX#125] [Progress Updates]
Can I delete branches f0004 and f0008?

**Comment by rvjansen** [comment]
F0004 can be deleted.

René.

**Reply by adesutherland** [reply]
Thanks - I can't remember what f0008 was for!

**Comment by rvjansen** [comment]
I have pushed a new branch feature/f0035 for completing the Rexx/RXAS bif library, mainly for Peter and myself.
